tests/AgentTests/SQLite.hs

```diff
@@ -34,11 +34,13 @@ storeTests = withStore do
     describe "createSndConn" testCreateSndConn
     describe "addSndQueue" testAddSndQueue
     describe "addRcvQueue" testAddRcvQueue
-    describe "deleteConnReceive" do
+    describe "deleteConn" do
       describe "Receive connection" testDeleteConnReceive
       describe "Send connection" testDeleteConnSend
-
--- describe "deleteConnDuplex" testDeleteConnDuplex
+      describe "Duplex connection" testDeleteConnDuplex
+    describe "updateQueueStatus" testUpdateQueueStatus
+    describe "updateQueueStatusBadDirectionSnd" testUpdateQueueStatusBadDirectionSnd
```

Comment:
Better to group under the method, so the title is either a method name or plain English